src/main/java/org/helpberkeley/memberdata/DriverPostFormat.java

```diff
@@ -118,6 +119,17 @@ protected abstract ProcessingReturnValue processLoopListRef(
     protected abstract String versionSpecificSimpleRef(MessageBlockContext context, String varName);
     protected abstract String versionSpecificDriverListRef(MessageBlockContext context, String varName);
 
+    public final void generateOpsManagerPhone() {
+        ControlBlock controlBlock = getControlBlock();
+        ControlBlock.OpsManager opsManager = controlBlock.getFirstOpsManager();
+        if (opsManager.getPhone().isEmpty()) {
```

Comment:
A small comment here might be useful.  Example:

// Support optional phone number in OpsManager macro